garde/src/rules/pattern.rs

```diff
@@ -69,6 +69,59 @@ impl<T: Pattern> Pattern for Option<T> {
     }
 }
 
+#[cfg(all(
+    feature = "regex",
+    all(feature = "js-sys", all(target_arch = "wasm32", target_os = "unknown"))
+))]
```

Comment:
what's with all the nesting? (no pun intended) isnt `all` just `&&`? so you could:
```suggestion
#[cfg(all(
    feature = "regex",
    feature = "js-sys",
    target_arch = "wasm32",
    target_os = "unknown",
))]
```

Reply:
same deal in other places

Reply:
Flattened all the expressions